docs/ml-features.md

```diff
@@ -1064,7 +1069,8 @@ categorical features.
 The bin ranges are chosen by taking a sample of the data and dividing it into roughly equal parts.
 The lower and upper bin bounds will be `-Infinity` and `+Infinity`, covering all real values.
 This attempts to find `numBuckets` partitions based on a sample of the given input data, but it may
-find fewer depending on the data sample values.
+find fewer depending on the data sample values. Relative precision of the approxQuantile is set using
+a relativeError parameter.Default value is 0.001. This parameter is not available in Python yet.
```

Comment:
I filed https://issues.apache.org/jira/browse/SPARK-15442 for the missing parameter - we should add that